docs/reference/metadata.md

```diff
@@ -214,15 +274,62 @@ Indentation options control aspects of the auto-indentation mechanism.
 
 `unIndentedLinePattern`
 : *Regex.*
-  The auto-indenter will ignore
-  lines matching this regex
-  when computing the next line's indentation level.
+  If it matches on the current line,
+  undo all the indent-level changes made by other options matching current line.  
+  Mostly used to except a particular line in the lines fuzzy matched by other options.
```

Comment:
There is no "fuzzy matching" going on here.

Also, I don't actually get this description and the examples that follow.